### ISSUE TITLE
clicking on micro App name (under the 'add object' icon) does not show the 'edit', 'trash', 'move' and 'rename' options

### ISSUE THREAD

**Issue by panditthecoder** [objecthub-io/ObjectHub#220]
after logging into the ide, if you click on the name of a micro app that has been created, it does not show the 'edit', 'trash', 'move' and 'rename' options. however, if you click on the icon to the left of the name, the icons are displayed.
see image for details
![no_edit_trash_move_rename_icon](https://user-images.githubusercontent.com/29138377/26923327-66368c60-4c5f-11e7-968d-c3ab7078639e.JPG)

**Comment by objecthub-io** [comment]
Thank you for reporting this issue!

This was indeed how we designed it to work. But you are correct, the behaviour here is a bit confusing. We'll discuss and get back to you!

**Comment by objecthub-io** [comment]
We have changed the functionality now that clicking on the icon next to the name of the object will also open the object.

Clicking anywhere else on the item will select it.

**Comment by panditthecoder** [comment]
so not sure if this change or the change for #221 caused it but if i delete and object and try to deploy it (click on the object and the code shows up on the editor side (which has the deploy button), click 'delete' on the right pane and deploy on the left pane), the 'edit', 'trash' 'rename' etc buttons vanish and do not show up irrespective of whether i click on another object or the icon next to the object. unfortunately multiple changes have been put back and so am not sure which putback is causing this set of issues.
bottom line, I am unable to see the 4 icons irrespective of where i click (service name or service icon). this was working for me earlier this evening.

**Comment by objecthub-io** [comment]
Does this issue persist after a reload of the page?

**Comment by panditthecoder** [comment]
OK. The behaviour on my chrome browser is now
- click on object icon - nothing* happens to the set of buttons (edit,trash etc) but the objects loads in the editor
- click on the object name - - nothing* happens to the set of buttons (edit,trash etc) but the objects loads in the editor
- click on any other part of the rectangle around the object - rectangle toggles between highlighted and not highlighted. when it is highlighted, the set of buttons (edit, trash et al) are displayed and when it is not highlighted, they are hidden. 
Seems to be a good compromise as the buttons clearly link to the item which is highlighted. so you KNOW that you are taking the specific action on that object.

PS : Nothing* means that if the buttons were visible, they continue to be visible and if they were not, they continue to not be visible

**Comment by objecthub-io** [comment]
Excellent! The behaviour you describe is exactly what we intended - and happy to hear you agree this is a good compromise.

Closing this issue now.
